src/test/java/step2/domain/LottoTest.java

```diff
@@ -0,0 +1,24 @@
+package step2.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.util.ReflectionTestUtils;
+
+class LottoTest {
+
+    @Test
+    public void setPrizeTest() {
+        //given
+        List<Integer> winningNumbers = Arrays.asList(1, 2, 3, 4, 5, 6);
+        Lotto lotto = new Lotto(winningNumbers);
+
+        //when
+        ReflectionTestUtils.invokeMethod(lotto, "setPrize", winningNumbers);
```

Comment:
`setPrize()` 의 access modifier는 public 이여서 굳이 relfection을 사용하지 않아도 되겠네요. 
가능하면 테스트 코드에서 ReflectionTestUtils 의존성을 줄여봤으면 좋겠어요. 
지금은 편리함보다는 더 좋은 코드를 위한 연습이 목적이니까요. 

```suggestion
        lotto.setPrize(winningNumbers);
```

그리고 1등 외 다른 등수에 대한 테스트 케이스가 함께 있으면 더 완벽한 테스트가 될 수 있지 않을까요?